libgeotiff/cmake/project-config.cmake.in

```diff
@@ -29,28 +24,28 @@ get_filename_component (_ROOT "${_DIR}/@PROJECT_ROOT_DIR@" ABSOLUTE)
 set (@PROJECT_NAME@_INCLUDE_DIRS "${_ROOT}/include")
 set (@PROJECT_NAME@_LIBRARY_DIRS "${_ROOT}/lib")
 set (@PROJECT_NAME@_BINARY_DIRS "${_ROOT}/bin")
+unset (_ROOT)
+unset (_DIR)
```

Comment:
When trying this with GDAL CMake builds, I get
```
-- GeoTIFF configuration, version 1.7.1
--   include directory: ${GeoTIFF_INCLUDE_DIRS}
--   ${GeoTIFF_LIBRARIES} set to shared library
CMake Error at /home/even/libgeotiff/install-cmake/share/cmake/GeoTIFF/geotiff-config.cmake:47 (include):
  include could not find load file:

    /geotiff-depends.cmake
Call Stack (most recent call first):
  cmake/helpers/CheckDependentLibraries.cmake:117 (find_package)
  cmake/helpers/CheckDependentLibraries.cmake:324 (gdal_check_package)
  gdal.cmake:215 (include)
  CMakeLists.txt:212 (include)
```

The unset(_DIR) should be moved after the include ("${_DIR}/@PROJECT_NAME_LOWER@-depends.cmake")

Reply:
Now using `CMAKE_CURRENT_LIST_DIR` instead.